src/DynamoCLI/Program.cs

```diff
@@ -21,11 +21,11 @@ static internal void Main(string[] args)
                 DynamoModel model;
                 if (!String.IsNullOrEmpty(cmdLineArgs.ASMPath))
                 {
-                    model = Dynamo.Applications.StartupUtils.MakeModel(true, cmdLineArgs.ASMPath);
+                    model = Dynamo.Applications.StartupUtils.MakeModel(true, cmdLineArgs.ASMPath, cmdLineArgs.AnalyticsInfo);
                 }
                 else
                 {
-                    model = Dynamo.Applications.StartupUtils.MakeModel(true);
+                    model = Dynamo.Applications.StartupUtils.MakeModel(true, string.Empty, cmdLineArgs.AnalyticsInfo);
```

Comment:
@QilongTang Hi Aaron.  Is this the correct way to update the `MakeModel` usage from the obsoleted methods

Reply:
yes